src/test/components/TrackScreenshots.test.js

```diff
@@ -137,12 +137,12 @@ describe('timeline/TrackScreenshots', function() {
   });
 
   it('is created in the <Timeline /> with a profile with screenshots', function() {
-    const { getByText } = setup(getScreenshotTrackProfile(), <Timeline />);
+    const { getAllByText } = setup(getScreenshotTrackProfile(), <Timeline />);
 
     // The function `getByText` throws already, with a useful Error, if it can't
     // find the element. But we still use `expect` to keep a "test-like"
     // assertion, even if it's useless.
```

Comment:
The comment looks outdated, can you please update it to mention `getAllByText` instead? `getAllByText` still throws if it finds nothing so the logic is very similar :)